### ISSUE TITLE
Filter a value of attribute table with a popup like in excel for a fast editing

### ISSUE THREAD

**Issue by atabaraud** [qgis/QGIS#37332]
Implement a pop up filter like in excel in the attribute table for quick filtering of values

Details of this fonctionality of excel :
![image](https://user-images.githubusercontent.com/38242880/85231423-21961780-b3f7-11ea-845e-94a13c5fc0e5.png)

How it can be implemented in QGIS :
![image](https://user-images.githubusercontent.com/38242880/85231546-39ba6680-b3f8-11ea-83d5-e267d92196f5.png)

**Comment by kaptainkory** [comment]
I agree and would find this kind of functionality very, very helpful.  For all of the fancy geospatial features added to QGIS it is kind of sad the attributes table remains so basic in functionality.  I think there is an incorrect assumption that much of the underlying data fed into a project will be pretty much "ready to go" as is.  I can tell you that for me, this couldn't be farther from the truth.  I've just spent the last several years working through georeferencing old biological records that came to me with very vague locality descriptions....where I needed to check records, often one-by-one, and readjust georeferencing to match.  Having a more functional attributes table that allowed for easier sorting and filtering across fields along with the ability to edit across rows/columns at the same time would have been VERY helpful.

I recommend looking at the implementation in this project: https://sqlitestudio.pl/

The ease and intuition of filtering by multiple columns and options for filtering are far superior to QGIS, albeit not quite the Excel way of doing things.  They also allow editing across rows and columns simultaneously, even with copy-and-paste!  They show changes in the data and even allow specific rollbacks on selected cells before commiting.

![Screenshot from 2020-06-22 19-53-57](https://user-images.githubusercontent.com/59102670/85348764-38ee0700-b4c2-11ea-971a-d39d59836ef3.png)

**Comment by kaptainkory** [comment]
Without making a separate issue, just SHOWING changes made to the data (e.g., highlighting adds as green, deletes as red) before commits alone would be a vast improvement to the attributes table.
